DOCS.md

```diff
@@ -100,7 +100,7 @@ publish:
 
 cache:
   mount:
-    - /drone/docker
+    - /var/lib/docker
```

Comment:
That will never work, we moved it to `/drone/docker` intentional because this is the only folder where the cache works.